client/components/checkbox/checkbox.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>CodePen - box_new</title>
+    <link rel="stylesheet" href="./checkbox.css">
+
+</head>
+
+<body>
+    <!-- partial:index.partial.html -->
+    <label class="container">
```

Comment:
`container` is a bit too "broad" of a class name for a label. Could you please change it to be more specific, like `checkbox-container` or something similar?